cmd/minikube/cmd/docker-env.go

```diff
@@ -50,6 +49,7 @@ import (
 	"k8s.io/minikube/pkg/minikube/reason"
 	"k8s.io/minikube/pkg/minikube/shell"
 	"k8s.io/minikube/pkg/minikube/sysinit"
+	kconst "k8s.io/minikube/third_party/kubeadm/app/constants"
```

Comment:
cc @neolit123

Reply:
the constants and features in use in minikube can technically be manually maintained in local constants and slices of features etc instead of being forked, but overall i don't see a big issue with keeping a small fork with a couple of files until kubeadm moves out.

the features package will likely be exposed publicly once that happens, but implementation details such as APICallRetryInterval are debatable, since they are more of an implementation detail - i.e. these could be constants/internal instead.

Reply:
I agree and thank you for the review :)